<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFG -&gt; Homsky</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="labelsForms">
      <item>
       <widget class="QLabel" name="lableCFG">
        <property name="text">
         <string>Приведенная КС-грамматика</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lableHomsky">
        <property name="text">
         <string>Нормальная форма Хомского</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QPlainTextEdit" name="rules">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="homskiyRules">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QListView" name="listCFG"/>
       <widget class="QListView" name="listHomskiy"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="errorLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 29, 21);</string>
      </property>
      <property name="text">
       <string>Error Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadConfigurationButton">
      <property name="text">
       <string>Загрузить конфигурацию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateHomskiy">
      <property name="text">
       <string>Преобразовать к форме Хомского</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showChains">
      <property name="text">
       <string>Вывод всех цепочек в диапозоне</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="left"/>
      </item>
      <item>
       <widget class="QSpinBox" name="right"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="checkEqualButton">
      <property name="text">
       <string>Проверить эквивалентность</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
